cypress/integration/signInForm_spec.js

```diff
@@ -0,0 +1,23 @@
+describe("SignIn Form", () => {
+  it("Check SignIn Form fields", () => {
+    cy.visit("/login");
+    cy.get("input[type=email]").should("be.visible");
```

Comment:
Related to my previous comment, I think this approach could work better:

```javascript
cy.get("input[placeholder=Email]").type('user@example.com');
```

To make this test suite useful to other project owners, we could maybe test locally against, let's say, the 10 most popular frontend implementations (this would require scripting), and only against https://demo.realworld.io/#/ for github actions

That way, we could try to find a common ground amongst the different projects.

Any thoughts @ChrisZie, @geromegrignon?

Reply:
I plan to update the specs to require the addition of proper selectors for v2 implementations.

Reply:
Cypress recommends using `data-cy` selectors so I agree about avoiding using classes, etc.

See the [Did you know?](https://docs.cypress.io/guides/references/best-practices#How-It-Works) section about the selector playground (for thinking about v2)

Regarding v1, I made my case but it' no big deal ;)